Content/OpenConfPath.md

```diff
@@ -0,0 +1,23 @@
+# Open Conf Path
+
+_OpenConfPath_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). It specifies a path pattern, for which Sandboxie will not apply sandboxing for registry keys. This lets sandboxed programs have direct access to update system settings _outside the sandbox_. This setting essentially _punches a hole_ in the sandbox, at a particular registry key location.
```

Comment:
```suggestion
_OpenConfPath_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v1.0.0 / 5.55.0. It specifies a path pattern, for which Sandboxie will not apply sandboxing for registry keys. This lets sandboxed programs have direct access to update system settings _outside the sandbox_. This setting essentially _punches a hole_ in the sandbox, at a particular registry key location.
```